Add airspeed docs for rover and copter (#4271)
ArduPilot/ardupilot_wiki: common/source/docs/common-airspeed-sensor.rst

.. _common-airspeed-sensor:

[copywiki destination="copter,rover"]

Airspeed Sensor
===============

> Copter, Rover, and Blimp can attach and enable an Airspeed Sensor. However, unlike Plane which uses the readings for navigation and control, these vehicles only log and report the readings for use in displays in Ground Control Stations, OSDs, and telemetry. The Plane airspeed sensor :ref:`documentation<plane:airspeed>` can followed for selecting and mounting the type of airspeed sensor, but be aware that it will not provide the functionality beyond measuring and reporting discussed in Plane.

Reviewer: ```suggestion
Copter, Rover, Sub, and Blimp can attach and enable an Airspeed Sensor. However, unlike Plane which uses the readings for navigation and control, these vehicles only log and report the readings for use in displays in Ground Control Stations, OSDs, and telemetry. The Plane airspeed sensor :ref:`documentation<plane:airspeed>` can followed for selecting and mounting the type of airspeed sensor, but be aware that it will not provide the functionality beyond measuring and reporting discussed in Plane.
```

Author: Actually Sub is an independently maintained Wiki...you wont find SUB stuff in the other vehicle's wiki's

> Copter, Rover, and Blimp can attach and enable an Airspeed Sensor. However, unlike Plane which uses the readings for navigation and control, these vehicles only log and report the readings for use in displays in Ground Control Stations, OSDs, and telemetry. The Plane airspeed sensor :ref:`documentation<plane:airspeed>` can followed for selecting and mounting the type of airspeed sensor, but be aware that it will not provide the functionality beyond measuring and reporting discussed in Plane.

Reviewer: ```suggestion
Copter, Rover, and Blimp can attach and enable an Airspeed Sensor. However, unlike Plane which uses the readings for navigation and control, these vehicles only log and report the readings for use by the displays in Ground Control Stations, OSDs, and telemetry. The Plane airspeed sensor page :ref:`documentation<plane:airspeed>` can be followed for selecting and mounting an airspeed sensor. However, it will not provide the functionality discussed in Plane except for measurement and reporting.
```

Author: modified similar to suggestion

The one exception to the above is that Sailboats can use an airspeed sensor as part of a :ref:`Wind Vane<rover:wind-vane>` (:ref:`WNDVN_TYPE<WNDVN_TYPE>` = 4) and use its readings to calculate wind speed as a part of navigation.

The following airspeed parameters have no effect and should always be set to "0" value:

- :ref:`ARSPD_USE<ARSPD_USE>`
- :ref:`ARSPD2_USE<ARSPD2_USE>`

> - :ref:`ARSPD_OPTIONS<ARSPD_OPTIONS>`

Reviewer: I think for the wiki I would add that you can still use Disable_Voltage_Correction.

Like I stated in the other PR though the others here could be made functional by setting USE and the option bit I don't think we should advertise them for now.

Author: so I dont intend to mention it....no real value at this point....if airspeed is actually used for something, we can change it then for that vehicle
